dags/map_reproducibility/utils.py

```diff
@@ -113,10 +115,9 @@ def wait_for_jobs_cmds():
 
 def copy_bucket_cmds():
   copy_bucket_contents = (
-      "COMPLETE_JOB_NAME=$(gcloud storage ls "
+      "export COMPLETE_JOB_NAME=$(gcloud storage ls "
       "gs://$BUCKET_NAME/nemo-experiments/ | grep $JOB_NAME)",
```

Comment:
nit: this is probably not blocking for now, for variables like this`nemo-experiments`, after we onboard more recipes, we should plan to refactor those variables defined in a central file, instead of copying and modifying the commands for each individual pipeline.

Reply:
Ack, will do that in next PR